docs/faq.md

```diff
@@ -159,6 +159,13 @@ be bridges between Polkadot and most of the other major chains.
 
 ## DOT
 
+### What is the difference between DOT (old) and new DOT?
+
+The DOT (old) unit on Polkadot was at twelve decimal places, otherwise known as 1e12 Plancks. On 21
+August, 2020, Demoniation Day, the DOT (old) value was redominated to 1e10 Plancks, meaning that the
+new DOT was valued at ten decimal places. Following the redomination, the new DOT changed back to
+DOT.
```

Comment:
It didn't "change back" to DOT, we started _calling_ it DOT instead of _New DOT_.